src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -231,7 +228,7 @@ export default class Result extends React.Component {
     };
 
     return (
-      <Box flex="1">
+      <ResultContainer innerRef={ref => (this.container = ref)}>
```

Comment:
What's this do?

Reply:
So it's `innerRef` instead of `ref` because styled components, and `this.container` so we can access this DOM node to get its height later